common/djangoapps/student/urls.py

```diff
@@ -3,65 +3,58 @@
 """
 
 from django.conf import settings
-from django.conf.urls import patterns, url
+from django.conf.urls import url
+from django.contrib.auth.views import password_reset_complete
 
-from student.views import LogoutView
+import student.views
 
-urlpatterns = (
-    'student.views',
-
-    url(r'^logout$', LogoutView.as_view(), name='logout'),
+urlpatterns = [
+    url(r'^logout$', student.views.LogoutView.as_view(), name='logout'),
 
     # TODO: standardize login
 
     # login endpoint used by cms.
-    url(r'^login_post$', 'login_user', name='login_post'),
+    url(r'^login_post$', student.views.login_user, name='login_post'),
     # login endpoints used by lms.
-    url(r'^login_ajax$', 'login_user', name="login"),
-    url(r'^login_ajax/(?P<error>[^/]*)$', 'login_user'),
+    url(r'^login_ajax$', student.views.login_user, name="login"),
+    url(r'^login_ajax/(?P<error>[^/]*)$', 'student.views.login_user'),
 
-    url(r'^email_confirm/(?P<key>[^/]*)$', 'confirm_email_change'),
+    url(r'^email_confirm/(?P<key>[^/]*)$', 'student.views.confirm_email_change'),
```

Comment:
This one and the one above it still need to be switched to callables.

Reply:
Should be fixed now